kubernetes/cache/proxy_status.go

```diff
@@ -41,7 +40,7 @@ func (c *kialiCacheImpl) pollIstiodForProxyStatus(ctx context.Context) {
 					interval := c.tokenNamespaceDuration / 2
 					retryErr := wait.PollImmediateUntilWithContext(ctx, interval, func(ctx context.Context) (bool, error) {
 						log.Trace("Getting proxy status from istiod")
-						proxyStatus, err = c.istioClient.GetProxyStatus()
+						proxyStatus, err = c.clientFactory.GetSAHomeClusterClient().GetProxyStatus()
```

Comment:
I think this is OK as we are starting with a single control plane. But as we add support to other models, this may need to evolve, as I think we won't be able to tie this proxy status cache to a cluster cache, nor the home cluster will be the only one having a control plane.

Reply:
Yes we'll need to scrape all of them and most likely merge the results together.